README.md

```diff
@@ -2,23 +2,46 @@
 
 The Boo Programming Language (c) 2009 Rodrigo B. de Oliveira (rbo@acm.org)
 
+Prerequisites
+=============
+
+## Mono
+
+If you are building on Mono, you need to have a compatible version from the 4.2.x line. The most recent compatible version is [4.2.4](https://download.mono-project.com/archive/4.2.4/).
+
+## Build Tools
+
+### Windows/Linux
+You'll need to install compatible versions of NAnt and NUnit. See [build-tools/versions](build-tools/versions) for known-good versions. _NAnt must be cloned and built from sources_; the last official release (0.92) does not work.
```

Comment:
The last official release (0.92) was released June 9, 2012, and did not support .NET 4.5. Attempting to build with a target of mono-4.5 results in an error message:

```
Invalid framework 'mono-4.5' specified.
```

I haven't checked but a similar error should occur when targeting net-4.5 on Windows.

Support for 4.5 was added 2015-10-22, but there has been no official release since then. No plans for a new official release have been announced. The last communication from a maintainer was in December 2016. For all intents and purposes, the NAnt project appears to be dead.